include/eve/algo/reduce.hpp

```diff
@@ -20,6 +20,40 @@
 
 namespace eve::algo
 {
+  //================================================================================================
+  //! @addtogroup algorithms
+  //! @{
+  //!  @var reduce
+  //!
+  //!  @brief SIMD version of std::reduce
```

Comment:
I would appreciate if the new documentation for algo follows the schema from the copy.hpp file with respect to :
 - sub header
 - global location for information
 - use of doxygen @note or @warning accordingly

 It'll svae time later down the road.

Reply:
Please recheck.